File: app/src/main/java/com/example/solar_alarm/Data/Tables/Timezone.java
package com.example.solar_alarm.Data.Tables;

import androidx.room.Entity;

@Entity(tableName = "Timezones")
public class Timezone extends TableBase
{
    public String CountryCode;

    public String CountryName;

    public String ZoneName;

    public String Abbreviation;

    public int GmtOffset;

    public Boolean Dst;

    public int ZoneStart;

    public int ZoneEnd;

    public String NextAbbreviation;

    public int Timestamp;
}

File: app/src/main/java/com/example/solar_alarm/Data/Tables/Timezone.kt
package com.example.solar_alarm.Data.Tables

import androidx.room.Entity
import androidx.room.Index
import com.example.solar_alarm.Data.Tables.TableBase

@Entity(tableName = "Timezone", indices = [Index(value = ["ZoneName"], unique = true)])
class Timezone : TableBase() {
    var CountryCode: String? = null
    var CountryName: String? = null
    var ZoneName: String? = null
    var Abbreviation: String? = null
    var GmtOffset = 0
    var Dst: Boolean? = null
    var ZoneStart = 0
    var ZoneEnd = 0
    var NextAbbreviation: String? = null
    var Timestamp = 0
}